src/components/modal/report/form-step.jsx

```diff
@@ -0,0 +1,89 @@
+const bindAll = require('lodash.bindall');
+const PropTypes = require('prop-types');
+const React = require('react');
+const FormattedMessage = require('react-intl').FormattedMessage;
+const classNames = require('classnames');
+
+const Form = require('../../forms/form.jsx');
```

Comment:
Ideally, we'd take this opportunity to implement what we need using Formik (the new forms system we're using instead of Formsy, which is passing into obsolescence).

An example is src/components/join-flow/country-step.jsx . We don't have an equivalent to the Formsy `Form` component, because we didn't think we needed one for Join Flow. 

We could make that change here, and I'd be happy to help. But I understand if you want to move faster than that would require. Just know that it's adding tech debt.

Reply:
Status: @paulkaplan and I will consider coming back to this to convert to Formik later